examples/headsail-bsp/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 
 pub mod sprintln;
 pub mod uart;
+pub mod timer_unit;
```

Comment:
One more thing! We might want to conditionally compile this line with `[cfg(feature = "vp")]` to make sure it's not included when targeting the ASIC.

Reply:
Ah, it's in the other PR. I would've put it in this one already but that's all fine. Consider this comment chain void.